.github/workflows/unit-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 15.x]
+        node-version: [14.x, 16.x, 17.x]
```

Comment:
In the b06 branch there were no changes related to the nodejs upgrade. The tests should pass for newer versions, but since we support nodejs 12 in this branch, maybe it's better to leave it?

Reply:
yeah, but I think that github stopped supporting it ? ...